src/crimson/os/cyanstore/cyan_store.h

```diff
@@ -35,6 +36,33 @@ class CyanStore final : public FuturizedStore {
   uuid_d osd_fsid;
 
 public:
+  class CyanOmapIterator final : public OmapIterator {
+    std::map<std::string, bufferlist>::const_iterator iter;
+    ObjectRef obj;
```

Comment:
could you put the private members at the end of the class? see https://google.github.io/styleguide/cppguide.html#Declaration_Order